core/tests/test_views.py

```diff
@@ -0,0 +1,659 @@
+from datetime import timedelta
+
+import pytest
+from django.urls import reverse
+from django.utils import timezone
+
+from core.models import (
+    Actor,
+    CinemaHall,
+    Genre,
+    Movie,
+    MovieScreening,
+    Seat,
+    Ticket,
+)
+from core.serializers import (
+    ActorSerializer,
+    CinemaHallSerializer,
+    GenreSerializer,
+    SeatSerializer,
+)
+from core.tests.factories import (
+    ActorFactory,
+    CinemaHallFactory,
+    GenreFactory,
+    MovieFactory,
+    MovieScreeningFactory,
+    SeatFactory,
+    TicketFactory,
+)
+
+
+@pytest.mark.django_db
+class TestGenre:
+    url_list = reverse("genre-list")
+
+    def test_genre_post(self, client):
+        genre = GenreFactory.build()
+        serialized = GenreSerializer(genre).data
+
+        response = client.post(
+            self.url_list, data=serialized, content_type="application/json"
+        )
+        response_data = response.json()
+
+        assert response.status_code == 201
+        assert response_data["name"] == serialized["name"]
+        assert response_data["is_for_adults"] == serialized["is_for_adults"]
+        assert Genre.objects.filter(name=serialized["name"]).exists()
```

Comment:
not sure if we have id in the model, if so, I would always default to checking by id, even if other fields are unique. Now they are, in the future they might not be, id/pk always id unique. tl;dr idiot proofing tests